test_sets/cloudApp_RPCs.txt

```diff
@@ -4,4 +4,5 @@
 ./test_scripts/CloudAppRPCs/004_CloudAppVehicleId_OnVehicleData.lua
 ./test_scripts/CloudAppRPCs/005_SetCloudAppProperties_Success.lua
 ./test_scripts/CloudAppRPCs/006_SetCloudAppProperties_InvalidParams.lua
-./test_scripts/CloudAppRPCs/007_SetCloudAppProperties_InitNewCloudApp.lua
\ No newline at end of file
+./test_scripts/CloudAppRPCs/007_SetCloudAppProperties_InitNewCloudApp.lua
+./test_scripts/CloudAppRPCs/008_UpdateAppList_NewCloudApplication.lua
```

Comment:
`008` is already in use for GetCloudAppProperties. `009` and `010` are also in use
```suggestion
./test_scripts/CloudAppRPCs/011_UpdateAppList_NewCloudApplication.lua
```